pkg/internal/testabilities/dbfixtures/db.go

```diff
@@ -48,3 +55,18 @@ func TestDatabase(t testing.TB) (db *database.Database, cleanup func()) {
 	require.NoError(t, err)
 	return db, func() {}
 }
+
+const sqliteFileNamePattern = `^file:(.+)\.sqlite(.*)$`
```

Comment:
would not be better to compile it once globally ? instead of doing this inside below func each time func is called ? 
```suggestion
var sqliteFileNameRegex = regexp.MustCompile(`^file:(.+)\.sqlite(.*)$`)
```

Reply:
Doesn't matter - this is just for tests.
(`testabilities`)

Reply:
ya but still i would use this once globally :)